docs/transition-phase-2.md

```diff
@@ -0,0 +1,107 @@
+# Transition your stake from phase-1 to phase-2 through CLI
+
+## Understanding the transition
+
+The transition from phase-1 to phase-2 is a manual process that requires the 
+staker to transition their stake to phase-2 of the Babylon network. 
+This transition has the option of being executed through the `stakercli` binary 
+and this guide will show you how to do it through the `stakercli` binary. For 
+detailed information about stake eligibility 
+criteria, registration requirements, and how to gain voting power and earn 
+rewards in phase-2, please refer to the 
+[Transition of Existing Stakes to the Babylon Chain](https://github.com/babylonlabs-io/babylon/blob/main/docs/stake-registration.md).
+<!-- TODO: Update link -->
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Setup](#setup)
+- [Transition your stake](#transition-your-stake)
+- [Verify the transition](#verify-the-transition)
+
+## Prerequisites
+
+To transition your stake from phase-1 to phase-2, you need to have the 
+following:
+- Access to your existing Bitcoin wallet from phase-1 (needed to sign transactions 
+  and prove ownership of staked funds).
+- The Bitcoin block height in which your staking transaction was included, which 
+  will be used in the transition command.
+- A funded Babylon address, which will be used to receive the rewards from the 
+  transition.
+- Global parameter file as seen here https://github.com/babylonlabs-io/networks/blob/main/bbn-test-5/parameters/global-params.json 
+  that were used in phase-1. This will be used in the transition command later 
+  on.
+- A running instance of the `stakercli` binary. Please follow the `stakercli` 
+  [setup guide](../README.md#3-btc-staker-installation) to complete the setup of 
+  the `stakercli` with your keys before proceeding.
+
+## Setup 
+
+First, you will need to dump the default configuration file. This creates a 
+template configuration file that you'll customize with your specific values 
+(like your Bitcoin block height that your staking transaction was included in, 
+wallet credentials, and network details):
+
+```bash
+stakercli admin dump-config
+```
+
+This will create a `config.json` file in your current working directory. 
+You'll need to modify this file with your specific settings before proceeding. 
+Here are the key configurations:
+
+### Bitcoin Configuration
+```toml
+[walletconfig]
+WalletName = "btc-staker"  # Your Bitcoin wallet name
+WalletPass = "your-wallet-password"  # Your wallet password
+
+[walletrpcconfig]
+Host = "127.0.0.1:19001"  # Your Bitcoin node RPC endpoint
+User = "your-rpc-user"    # Your Bitcoin RPC username
+Pass = "your-rpc-pass"    # Your Bitcoin RPC password
+
+[chain]
+Network = "signet"  # Bitcoin network (signet for testnet)
+```
+
+### Babylon Configuration
+```toml
+[babylon]
+Key = "btc-staker"                    # Your Babylon key name
```

Comment:
Do we need to instruct user how to create this key?

Reply:
Looks like it's here https://github.com/babylonlabs-io/btc-staker/blob/6d25cf181be58531fff2ed32d2b0625887b9ff6a/README.md#create-a-babylon-keyring-keyring-backend-test-with-funds